host_vars/manstein10-m2-s/base.yml

```diff
@@ -3,3 +3,5 @@
 location: manstein10
 role: ap
 model: "ubnt_nanostation-m2_xm"
+
+wireless_profile: freifunk_default
```

Comment:
that should be unnecessary?

Reply:
maybe. But better be explicit than implicit. :)

Reply:
Just removed it.